src/components/EntityInfo/EntityMenu.vue

```diff
@@ -314,6 +365,10 @@ export default class EntityMenu extends Mixins(AllowableActionsMixin) {
     this.emitConfirmDissolution()
   }
 
+  goToAmalgamationSelection (): void {
+    this.$router.push({ name: Routes.AMALGAMATION_SELECTION, params: { filingId: '0' } })
```

Comment:
You don't need params for your new page/view. It's used for the local filings to denote a new filing and to prevent users from going right to that route (since filingId would be undefined).

Reply:
In retrospect, I think you _should_ use this (and check for filingId===0 in created) to prevent users from bypassing the entity info header button and going straight to the selection page.

Reply:
Fixed, thanks.